### PR TITLE
chore(vendor): bump openhuman and tinyhivemind to upstream main

### PR DESCRIPTION
**Pull request by senamakel**
Bumps both vendored submodules to their upstream `main`.

| Submodule | From | To |
|---|---|---|
| `vendor/openhuman` | `60eeded16` | `9404515cb` (v0.63.24) |
| `vendor/tinyhivemind` | `2eba18118` | `57281e351` (#35, utterance-surface) |

`Cargo.lock` follows with `openhuman 0.63.23 -> 0.63.24`. That is the whole diff — three lines.

## Commands run locally (aarch64-darwin)

| Suite | Result |
|---|---|
| `cargo test` | 4997 passed, 0 failed, 7 ignored |
| `frontend` `npm test` (vitest) | 5176 passed, 3 skipped, 597 files |
| `npm run e2e` (default host) | 383 passed, 0 failed, 34 skipped |
| `npm run e2e:first-run` | 2 passed |
| `npm run e2e:live` (`--features openhuman,mcp,composio`) | 395 passed, 0 failed |
| `npm run e2e:live-llm` | 1 passed |

Both host binaries were built per the CI lanes (`cargo build --locked --bin opencompany`
and `cargo build --locked --features openhuman,mcp,composio --bin opencompany`), and each
lane ran against its own binary on its own port.

## Notes for the reviewer

**`e2e:live-llm` needs a rung above `flash`.** On the proxy's default rung the spec failed
twice at different assertions — once never calling `delegate_to_desk` at all, once never
reaching `done` inside the 300s budget. On `PW_LIVE_LLM_MODEL=max-reasoning` it passes in
4.0m: the model delegates, the desk writes the artifact, and the orchestrator closes the
task out. Since that lane's verdict is a model's judgement rather than the code's, the
`flash` failures look like rung capability and not a regression from this bump — but the
default in `live-brain-proxy.mjs` is worth a second opinion.

**The visual lane was not run for this PR.** It cannot pass on darwin as the repo stands:
`frontend/test/e2e/visual.spec.ts-snapshots/` ships `*-linux.png` for every surface but has
no `-darwin` variant for `workflows` or `overview`, so those tests fail with "a snapshot
doesn't exist" regardless of the change under test. Recording darwin baselines is out of
scope here.

Separately, and independent of platform: `overview renders as recorded` fails its
precondition rather than a pixel comparison — `getByRole('img', { name: 'Operating
knowledge graph' })` is never visible, timing out at 30s in both themes. Filed here as an
observation only; nothing in this bump touches it.

## Untested edge cases

No behaviour is added by this change, so no new tests accompany it. The suites above are
the existing coverage re-run against the new pins.

## Summary by CodeRabbit

* **Chores**
  * Updated bundled component versions to include the latest upstream changes.

### CODE REVIEW

File: vendor/tinyhivemind [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 2eba18118dc72ef7c2ea27c9187e41b972b3eb49
+Subproject commit 57281e351fad849a7e8656a27e35eb8567e1d248
```
[the remark this answers is not in the document]
Comment (senamakel) [reply vendor/tinyhivemind:1]: Ran TinyHiveMind's own regression suite at the pinned SHA (57281e351fad849a7e8656a27e35eb8567e1d248): `cargo test --manifest-path vendor/tinyhivemind/Cargo.toml --all-features` — 39 tests passed, 0 failed, across tinyhivemind, tinyhivemind-core, and tinyhivemind-hive (unit, integration, doc-tests, and the bench example's unit tests). The new `tests/utterance_surface.rs` specifically: 2 passed, 0 failed (an_aside_that_outruns_its_budget_falls_back_to_the_room, a_desk_that_works_a_problem_and_closes_it_commits_exactly_these_rows). No regressions found; this is a clean submodule bump.

**Comment by senamakel** [comment]
### PR babysitter status

**Head:** `7cdff3d49a0044903a5f52626849994503cf81cd`
**State:** All required CI checks green, no unresolved review threads, mergeable.

**Fixes pushed during babysitting:**
- `2710f1a70` — regenerated `src-tauri/Cargo.lock` (`cargo update -p openhuman`) so the desktop workspace's separate lockfile picks up openhuman 0.63.23 -> 0.63.24, matching the root `Cargo.lock`. The desktop CI lane runs `cargo clippy --locked` against this file, which was failing with "cannot update the lock file ... because --locked was passed" before this fix.
- `7cdff3d49` — dropped incidental unrelated lockfile churn (transitive `windows-sys` version drift) from the desktop refresh so the commit stays scoped to the openhuman bump.

**Validation performed:**
- `cargo metadata --manifest-path src-tauri/Cargo.toml --locked` — succeeds against the current head.
- `cargo clippy --manifest-path src-tauri/Cargo.toml --locked --all-targets --no-deps --features opencompany/acp,opencompany/composio -- -D warnings` — passes locally (exit 0).
- `cargo test --manifest-path vendor/tinyhivemind/Cargo.toml --all-features` at the pinned submodule SHA `57281e351fad849a7e8656a27e35eb8567e1d248` — 39 passed, 0 failed (includes the new `tests/utterance_surface.rs`: 2 passed, 0 failed), per CodeRabbit's request.

**CI (head `7cdff3d49`):** all required checks SUCCESS (Rust, Rust (mongodb), Rust (mail), Rust (openhuman, tinymemory), Desktop, Gated host binary, Console E2E, Console E2E (first run), Console E2E (live brain), Changed areas, Markdown line cap, Actions pinned). Console / Console (current Node, advisory) SKIPPED (no frontend changes). CodeRabbit and Codex (code + security) both completed with no findings at this head.

**Review feedback:** CodeRabbit's one actionable comment (run TinyHiveMind's own regression suite at the pinned SHA before merging) was addressed and replied to in-thread with the test evidence above; thread resolved. No other actionable feedback outstanding.

Ready for `pr-approval-reviewer`.
